lib/records/fhir.rb

```diff
@@ -240,6 +241,84 @@ def self.immunization(imm, fhir_record, patient, encounter)
         fhir_record.entry << entry
       end
 
+      def self.careplans(plan, fhir_record, patient, encounter)
+        careplanData = CAREPLAN_LOOKUP[plan['type']]
+        reasonCode = COND_LOOKUP[plan['reason']][:codes]['SNOMED-CT'][0]
+        reason = fhir_record.entry.find{|e| e.resource.is_a?(FHIR::Condition) && e.resource.code.coding.find{|c|c.code==reasonCode} }
+        careplan = FHIR::CarePlan.new({
+          'subject' => {'reference'=> "Patient/#{patient.fullUrl}"},
```

Comment:
Typically (in FHIR doc anyway), a UID only field has something like `urn:uuid:` as a prefix.  I think that the "full" part in "fullUrl" is meant to hint that at least a url scheme should be included.